app/assets/javascripts/views/story_view.js

```diff
@@ -463,14 +466,22 @@ module.exports = FormView.extend({
       new Clipboard('.btn-clipboard');
     }
 
-    const descriptionContainer = this.$('.description-wrapper')[0];
```

Comment:
Instead of using class selectors, what about using attribute selectors? You can have much more control over your DOM by doing `$('[some-attr]')` instead of `$('.some-class]')` because you can use values to the attributes to fine tune your needs.